layers/mos/stacker.yaml

```diff
@@ -23,10 +25,10 @@ mos-build:
     cp /stacker/zot-config.json "$workd/etc/"
 
     mkdir -p "$workd/usr/bin"
-    for bin in mosctl:mosctl zot-linux-amd64-minimal:zot ; do
```

Comment:
But if the user is specifying their own zot, they will likely just have /some/path/zot , and this won't handle that.

Can you either just try both, or do a glob?

Reply:
yeah, took your suggestion of importing to its own directory and using glob.